index.html

```diff
@@ -208,34 +335,77 @@ <h3><a class="textdecoration font1" href="#">Multi-Post Stories</a></h3>
 
             </div>
 
-        </article>
+            <div class="article4div show2">
+
+                <h3><a class="textdecoration font1" href="#">Uber Navigation</a></h3>
+
+                <ul class="ul1">
+
+                    <li><a class="textdecoration font2" href="#">CANOPY</a></li>
+                    <li><a class="textdecoration font2" href="#"><img src="./img/mid_dot.jpg" alt=" "></a></li>
+                    <li><a class="textdecoration font3" href="#">Back End Dev</a></li>
+                    <li><a class="textdecoration font2" href="#"><img src="./img/mid_dot.jpg" alt=" "></a></li>
+                    <li><a class="textdecoration font3" href="#">2015</a></li>
+
+                </ul>
+
+                <p>A smart assistant to make driving more safe, efficient, and fun by unlocking your most expensive
+                    computer: your car.</p>
+
+                <ul class="gridbox1">
+
+                    <li class="background1"><a class="textdecoration font4" href="#">html</a></li>
+                    <li class="background1"><a class="textdecoration font4" href="#">Ruby on rails</a></li>
+                    <li class="background1"><a class="textdecoration font4" href="#">css</a></li>
+                    <li class="background1"><a class="textdecoration font4" href="#">javaScript</a></li>
+
+                </ul>
+
+                <button type="submit"><a class="font5 textdecoration" href="#">See Project</a></button>
 
+            </div>
+
+            <figure class="figure2">
+
+                <a href="#"><img src="./img/jumbo2b.png" alt="jumbo4_img"></a>
+
+            </figure>
+
+        </article>
 
     </section>
 
-    <section class="about_self_section">
+    <section class="about_self_section bg_color7">
```

Comment:
Nice job so far on the about section. I think some small details are missing when compared to the Figma template. ✍️
- [ ] Kindly use `40%` width on the left section and `60%` on the right one to achieve the Design.
- [ ] Kindly increase the width of the cards that hold HTML, CSS, and Javascript icons and make the icons align to the left inside the card. Please update the text `color` below the icon images.
- [ ] I think a fixed with is being applied to the paragraph in the left section. Kindly update the `font-size` and remove the fixed width so the paragraph can span the whole parent element.

- Your Design
![Screenshot from 2022-12-08 20-30-52](https://user-images.githubusercontent.com/17243325/206535271-45f13f26-e446-4f06-bd03-5e91586095bd.png)

- Figma Design
![Screenshot from 2022-12-08 20-30-13](https://user-images.githubusercontent.com/17243325/206535405-24351987-2c02-4d75-a1ee-85a04da60a0f.png)